src/layouts/components/Editable/Editable.tsx

```diff
@@ -71,7 +71,13 @@ export const EmptySection = ({
       justifyContent="center"
     >
       {image}
-      <Text>{title}</Text>
+      <Text
+        textColor="base.content.strong"
+        textAlign="center"
+        lineHeight="1.75rem"
+      >
+        {title}
+      </Text>
```

Comment:
assumed to be cleared with design

Reply:
Partially yes, was updated in 5ae1776da0d577e004a2653cec54912e666d40b3.